runtime/common/src/precompile/schedule_call.rs

```diff
@@ -99,46 +77,36 @@ type PalletBalanceOf<T> =
 type NegativeImbalanceOf<T> =
 	<<T as module_evm::Config>::Currency as Currency<<T as frame_system::Config>::AccountId>>::NegativeImbalance;
 
-impl<
-		AccountId,
-		AddressMapping,
-		Erc20InfoMapping,
-		Scheduler,
-		ChargeTransactionPayment,
-		Call,
-		Origin,
-		PalletsOrigin,
-		Runtime,
-	> Precompile
-	for ScheduleCallPrecompile<
-		AccountId,
-		AddressMapping,
-		Erc20InfoMapping,
-		Scheduler,
-		ChargeTransactionPayment,
-		Call,
-		Origin,
-		PalletsOrigin,
-		Runtime,
-	> where
-	AccountId: Debug + Clone,
-	AddressMapping: AddressMappingT<AccountId>,
-	Erc20InfoMapping: Erc20InfoMappingT,
-	Scheduler: ScheduleNamed<BlockNumber, Call, PalletsOrigin, Address = TaskAddress<BlockNumber>>,
-	ChargeTransactionPayment: TransactionPayment<AccountId, PalletBalanceOf<Runtime>, NegativeImbalanceOf<Runtime>>,
-	Call: Dispatchable<Origin = Origin> + Debug + From<module_evm::Call<Runtime>>,
-	Origin: IsType<<Runtime as frame_system::Config>::Origin>
-		+ OriginTrait<AccountId = AccountId, PalletsOrigin = PalletsOrigin>,
-	PalletsOrigin: Into<<Runtime as frame_system::Config>::Origin> + From<frame_system::RawOrigin<AccountId>> + Clone,
-	Runtime: module_evm::Config + frame_system::Config<AccountId = AccountId>,
+impl<Runtime> Precompile for ScheduleCallPrecompile<Runtime>
+where
 	PalletBalanceOf<Runtime>: IsType<Balance>,
+	module_transaction_payment::ChargeTransactionPayment<Runtime>:
+		TransactionPayment<Runtime::AccountId, PalletBalanceOf<Runtime>, NegativeImbalanceOf<Runtime>>,
+	Runtime: module_evm::Config
+		+ module_prices::Config
+		+ module_transaction_payment::Config
+		+ pallet_scheduler::Config
+		+ Send
+		+ Sync,
+	<Runtime as pallet_scheduler::Config>::Call: Dispatchable + Debug + From<module_evm::Call<Runtime>>,
+	<<Runtime as pallet_scheduler::Config>::Call as Dispatchable>::Origin: IsType<<Runtime as frame_system::Config>::Origin>
+		+ OriginTrait<
+			AccountId = Runtime::AccountId,
+			PalletsOrigin = <Runtime as pallet_scheduler::Config>::PalletsOrigin,
+		>,
+	pallet_scheduler::Pallet<Runtime>: ScheduleNamed<
+		BlockNumber,
+		<Runtime as pallet_scheduler::Config>::Call,
```

Comment:
both `<Runtime as pallet_scheduler::Config>::Call` and `<Runtime as frame_system::Config>::Call`  works here, which one is better, or just doesn't matter because they use the same `Call`.

Reply:
this is fine